lbrynet/dht/protocol.py

```diff
@@ -24,6 +24,14 @@
 
 log = logging.getLogger(__name__)
 
+REMOTE_EXCEPTIONS = {
+    "exceptions.%s" % e: getattr(exceptions, e) for e in dir(exceptions) if not e.startswith("_")
+}
+
```

Comment:
This a list of built in exceptions no? 
Kind of confusing , may be rename it to BUILTIN_EXCEPTIONS